Applications/VLMEval/ContentView.swift

```diff
@@ -328,8 +331,8 @@ class VLMEvaluator {
     let modelConfiguration = ModelRegistry.smolvlm
 
     /// parameters controlling the output
-    let generateParameters = MLXLMCommon.GenerateParameters(temperature: 0.6)
-    let maxTokens = 800
+    let generateParameters = MLXLMCommon.GenerateParameters(temperature: 0.7, topP: 0.9)
+    let maxTokens = 400
```

Comment:
This was for testing, reverting
```suggestion
    let maxTokens = 800
```